src/linux/cgroups2.cpp

```diff
@@ -265,7 +265,7 @@ Try<Nothing> destroy(const string& cgroup)
     return Error("There does not exist a cgroup at '" + absolutePath + "'");
   }
 
-  Try<Nothing> rmdir = os::rmdir(absolutePath);
+  Try<Nothing> rmdir = os::rmdir(absolutePath, false);
```

Comment:
This PR has two commits. This change (and only this change) is from the first commit, where I explain why this was introduced. TLDR: make `os::rmdir` use `rmdir` instead of `rm`.